data/modules/Scout/Scout.lua

```diff
@@ -303,24 +303,30 @@ local function calcOrbitalScanMission(sBody, difficulty, reward)
 
 	local resolutionScalar = math.invlerp(p.resolution_min, p.resolution_max, resolution)
 
+	local body_coverage = coverage
 	if radiusScalar > 1.0 then
 		-- body is small, increase coverage of the scan
-		coverage = math.min(coverage, 1.0) * (radiusScalar ^ 0.9)
+		body_coverage = math.min(coverage, 1.0) * (radiusScalar ^ 0.9)
 	else
 		-- body is large, reduce coverage of the scan proportionally
-		coverage = math.min(coverage, 1.0) * radiusScalar
+		body_coverage = math.min(coverage, 1.0) * radiusScalar
 		-- similarly increase the resolution of the scan to ensure we can scan at higher altitudes
 		resolution = resolution * (1.0 + math.log(radiusKm / p.nominal_radius, 10))
 	end
 
-	coverage = math.min(coverage * realDifficulty, 1.0)
+	-- adjust coverage against difficulty and clamp to a maximum of 100%
+	body_coverage = math.min(body_coverage * realDifficulty, 1.0)
+
+	-- finally, convert the coverage into km2 and limit the number of decimals
+	local body_area = 4 * math.pi * radiusKm^2
+	local coverageKm2 = utils.round(body_area, 0.0001)
```

Comment:
This line is computing the mission's coverage as the entirety of the body's surface area, regardless of the `body_coverage` value above. Multiply `body_area` by `body_coverage` and we're good to go.

Reply:
Fixed in 0ba8f053b60e217f44971e4ee53d5565eff2776e

This was originally correct, but got lost in a refactor to use "utils.round()"..